src/components/Shared/Filters/NullState/NullState.test.tsx

```diff
@@ -1,19 +1,56 @@
 import React from 'react';
+import { MockedProvider } from '@apollo/client/testing';
 import { ThemeProvider } from '@mui/material/styles';
 import userEvent from '@testing-library/user-event';
 import { SnackbarProvider } from 'notistack';
-import TestWrapper from '__tests__/util/TestWrapper';
+import TestRouter from '__tests__/util/TestRouter';
 import { render, waitFor } from '__tests__/util/testingLibraryReactMock';
 import { TaskModalEnum } from 'src/components/Task/Modal/TaskModal';
+import { UrlFiltersProvider } from 'src/components/common/UrlFiltersProvider/UrlFiltersProvider';
 import useTaskModal from '../../../../hooks/useTaskModal';
 import theme from '../../../../theme';
 import NullState from './NullState';
 
-const changeFilters = jest.fn();
 const openTaskModal = jest.fn();
 
 jest.mock('src/hooks/useTaskModal');
 
+interface TestComponentProps {
+  page: 'contact' | 'task';
+  totalCount: number;
+  filters?: string;
+}
+
+const defaultRouter = {
+  query: {},
+  isReady: true,
+};
+
+const TestComponent: React.FC<TestComponentProps> = ({
+  page,
+  totalCount,
+  filters,
+}) => (
+  <TestRouter
+    router={{
+      ...defaultRouter,
+      query: {
+        filters,
+      },
+    }}
+  >
+    <SnackbarProvider>
+      <ThemeProvider theme={theme}>
+        <MockedProvider>
```

Comment:
Is there a reason to use `MockedProvider`? I always prefer `GqlMockedProvider` because it's more powerful since it does automatic mocking.

```suggestion
        <GqlMockedProvider >
```